php/router.php

```diff
@@ -0,0 +1,105 @@
+<?php
+// Used by `wp server` to route requests.
```

Comment:
They are not top-level, but inside the `WP_CLI\Router` namespace.

If you have specific questions that can't be answered by the current comments, I'd be glad to answer them.